client/src/app/+login/login.component.html

```diff
@@ -107,7 +111,7 @@ <h4 i18n class="modal-title">Forgot your password</h4>
     >
 
     <input
-      type="submit" i18n-value value="Send me an email to reset my password" class="action-button-submit"
+      type="submit" i18n-value value="Recover password" class="action-button-submit"
```

Comment:
```suggestion
      type="submit" i18n-value value="Reset my password" class="action-button-submit"
```

There is no way to _recover_ a password.

Reply:
Thanks @rigelk maybe only « Reset » because for languages like french «Réinitialiser mon mot de passe  » it won't pass.

Reply:
```suggestion
      type="submit" i18n-value="Password reset button" value="Reset" class="action-button-submit"
```